src/__swaps__/screens/Swap/components/SwapSlider.tsx

```diff
@@ -144,17 +145,31 @@ export const SwapSlider = ({
   });
 
   const onSlide = useAnimatedGestureHandler({
-    onStart: (_, ctx: { startX: number }) => {
+    onStart: (_, ctx: { exceedsMax?: boolean; startX: number }) => {
+      ctx.exceedsMax = undefined;
       ctx.startX = sliderXPosition.value;
       sliderPressProgress.value = withSpring(1, SPRING_CONFIGS.sliderConfig);
       SwapInputController.inputMethod.value = 'slider';
 
+      if (ctx.startX >= width) {
+        const currentInputValue = SwapInputController.inputValues.value.inputAmount;
+        const maxSwappableAmount = internalSelectedInputAsset.value?.maxSwappableAmount;
+        const exceedsMax = maxSwappableAmount ? greaterThanWorklet(currentInputValue, maxSwappableAmount) : false;
```

Comment:
this is fine but i think `maxSwappableAmount` always exists https://github.com/rainbow-me/rainbow/blob/@christian/fix-swap-max-amount/src/__swaps__/types/assets.ts#L22

Reply:
Can be undefined if the asset is null